frontend/src/components/pages/events/EventFields/RequiredFields.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Box, FormControl, Grid, InputLabel, MenuItem, Select, TextField, Typography } from "@material-ui/core";
+import { User } from "@interfaces/users";
+
+/**
+ * Component for filling in required fields on an event
+ */
+
+type Props = {
+  eventData: Record<string, any>;
+  onEventDataChange: (data: Record<string, any>) => void;
+  user?: User;
+};
+
+const RequiredFields: React.FC<Props> = ({ eventData, onEventDataChange, user }) => {
+  return (
+    <>
+      <Box paddingTop={3}>
+        <Typography variant="h4">Påkrevde felt</Typography>
+      </Box>
+      <Grid container spacing={3}>
+        <Grid item xs={6}>
+          <TextField
+            label="Tittel"
+            placeholder="Tittel"
+            value={eventData.title}
+            onChange={(e) => onEventDataChange({ ...eventData, title: e.currentTarget.value })}
```

Comment:
Both `title` and `startTime` are required in order to make a new event. I suggest passing a `required` prop to the corresponding textfields to make this extra clear
```suggestion
            onChange={(e) => onEventDataChange({ ...eventData, title: e.currentTarget.value })}
            required
```